docs/fieldnorm/user_feedback.md

```diff
@@ -0,0 +1,21 @@
+# Laboratory patients, user, and personnel feedback
+
+Bioinformaticians develop data analysis workflows, tools and (web) applications that are used by different groups of laboratory analists and specialists, from hereafter refered as 'end users'. These end users will experience events where unexpected, incorrect or occasionally no results are produced, due to bugs which are caused by either errors, flaws or faults in the design, development, or operation of the software developed by the bioinformaticians. In addition, end users may come up with requests and suggestions for new features / functionality to further improve or expand the scope of existing software. 
+
+The bioinformatics team therefore needs to setup a system through which both reports of issues / bugs and requests can be submitted, traced and processed. Rather than using a single procedure / system to handle both issues and new requests or feedback, different procedures may be set up to process and handle them independently in a different way. When the issues have been handled and actions have been taken, results need to be communicated back to the end users. 
+
+The requirements of the system are :
+
+-   Feedback / requests from end users can be submitted, registered and traced by the system. There is a documented procedure for end users on how to use the system to submit feedback / requests and how they will be processed. 
+-   Feedback / requests are evaluated and an evaluation is made as to which actions are needed and which requirements need to be met.  
+-   Actions that are being taken to process the feedback / requests are recorded and can be traced back to it either by the system itself or by reference to other records and documentation. 
```

Comment:
"Actions that are being taken to process the feedback / requests are recorded and can be traced back to" should be enough in my opinion. the last part of the sentence acts as an example but feels like it narrows it down. I would just leave it out.